raiden/tests/integration/api/test_pythonapi_regression.py

```diff
@@ -25,7 +25,7 @@ def test_close_regression(raiden_network, deposit, token_addresses):
     channel_list = api1.get_channel_list(registry_address, token_address, app1.raiden.address)
     channel12 = channel_list[0]
 
-    token_proxy = app0.raiden.proxy_manager.token(token_address)
+    token_proxy = app0.raiden.proxy_manager.token(token_address, "latest")
```

Comment:
Can we be sure here that `node1_balance_before` is equal to the internal state of `appX`, because that is a number of blocks behind?

Reply:
the `balance_of` call is a proxy call directly to the contracts. In it's definition, the block_identifier it takes is a default argument with a value of `latest` so in this test it is guaranteed to have the value we expect.

Reply:
Sure, then:
- Why is there a default value? Shouldn't we remove that?
- Why use "latest" here instead of `views.state_from_raiden(self.raiden).block_hash`

Reply:
Fixing the tests for these is out of scope of this PR... i think it's going to be alot of work.

Reply:
For the record, this already has an issue: https://github.com/raiden-network/raiden/issues/3619

Reply:
Taht's ok with me, but ca you create an issue for it.